### ISSUE TITLE
add training considerations to README

### ISSUE THREAD

**Issue by jettjaniak** [delphi-suite/delphi#20]
batch size & gradient accumulation
what constitutes the epoch, number of epochs
spacing between checkpoints
